templates/drupal8/files/.platform.app.yaml

```diff
@@ -27,16 +27,26 @@ runtime:
 # The size of the persistent disk of the application (in MB).
 disk: 2048
 
-# The 'mounts' describe writable, persistent filesystem mounts in the application. The keys are
-# directory paths, relative to the application root. The values are strings such as
-# 'shared:files/PATH', where PATH is a relative path under the mount's source directory.
+# The 'mounts' describe writable, persistent filesystem mounts in the application.
 mounts:
-    '/web/sites/default/files': 'shared:files/files'
-    '/tmp': 'shared:files/tmp'
-    '/private': 'shared:files/private'
-    '/.drush': 'shared:files/.drush'
-    '/drush-backups': 'shared:files/drush-backups'
-    '/.console': 'shared:files/console'
+    '/web/sites/default/files':
+        source: local
+        source_path: 'files'
+    '/tmp':
+        source: local
+        source_path: 'tmp'
+    '/private':
+        source: local
+        source_path: 'private'
+    '/.drush':
+        source: local
+        source_path: 'drush'
```

Comment:
```suggestion
        source_path: '.drush'
```

Reply:
I'm not sure if the dot prefix here is even wise.  It shouldn't matter to the code, but making the mounted directly semi-private doesn't seem like it offers any advantage, does it?

Reply:
Not to me no, just matching what was there. No reason to make them hidden in my mind.